### PR TITLE
Implementing Isogram

### PR DESCRIPTION
**Pull request by Average-user**

### CODE REVIEW

File: exercises/isogram/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Examples of isograms:
+
+- lumberjacks
+- background
+- downstream
+- six-year-old
+
+The word *isograms*, however, is not an isogram, because the s repeats.
+
+If everything goes well, you will be ready to fetch your first real exercise.
```
Comment (stevejb71) [inline exercises/isogram/README.md]: Is this a copy from elsewhere?

Comment (Average-user) [reply exercises/isogram/README.md]: from [https://github.com/exercism/problem-specifications/blob/master/exercises/isogram/description.md](https://github.com/exercism/problem-specifications/blob/master/exercises/isogram/description.md)

Comment (Average-user) [reply exercises/isogram/README.md]: Ohh, probably from hello world, I Should remove it

**Comment by yurrriq** [comment]
Now that we're using [idris-testing](https://github.com/yurrriq/idris-testing) could you rewrite the tests accordingly? Otherwise this looks good.

**Comment by Average-user** [comment]
Sure I'll do it

**Pull request closed without merging**